setup.py

```diff
@@ -202,6 +202,11 @@ def run(self):
     :undoc-members:
 """ % mod)
 
+# (function, predicate), see http://docs.python.org/2/distutils/apiref.html#distutils.cmd.Command.sub_commands
```

Comment:
it doesn't make much sense to automatically trigger (and thus require) the usage/api.rst build (and require mock additionally) for most installs as most installs won't build the html docs locally from that anyway (which would need even more dependencies pulled in via sphinx).